libs/application/ui-fields/src/lib/AsyncSelectFormField.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC, useEffect, useState } from 'react'
+import {
+  AsyncSelectField,
+  FieldBaseProps,
+  formatText,
+} from '@island.is/application/core'
+import { Box } from '@island.is/island-ui/core'
+import {
+  SelectController,
+  FieldDescription,
+} from '@island.is/shared/form-fields'
+import { useLocale } from '@island.is/localization'
+import { useApolloClient } from '@apollo/client/react'
+import { Option } from '@island.is/application/core'
+
+interface Props extends FieldBaseProps {
+  field: AsyncSelectField
+}
+const AsyncSelectFormField: FC<Props> = ({ application, error, field }) => {
+  const { id, name, description, loadOptions, placeholder, disabled } = field
+  const { formatMessage } = useLocale()
+  const apolloClient = useApolloClient()
+  const [options, setOptions] = useState<Option[]>([])
+
+  useEffect(() => {
+    async function load() {
+      const loaded = await loadOptions({ application, apolloClient })
+      setOptions(loaded)
+    }
+
+    load()
+  }, [])
```

Comment:
you probably have to add the dependencies into the array or?

Reply:
Well...I just want it to run in the first render, but I guess the loadOptions func should be a dep ?

Reply:
I just thought the linter would complain